libevm/VMCalls.cpp

```diff
@@ -120,6 +120,9 @@ void VM::caseCreate()
 	uint64_t initOff = (uint64_t)m_SP[1];
 	uint64_t initSize = (uint64_t)m_SP[2];
 
+	// Clear the return data buffer. This will not free the memory.
+	m_returnData.clear();
+
```

Comment:
See the last sentence on this line: https://github.com/ethereum/EIPs/pull/211/files#diff-fd8c393953fe66162d8bb518d87bfb57R32

(That's something I learned while doing YP.)